packages/components/card/src/base-card/BaseCard.tsx

```diff
@@ -66,7 +66,6 @@ function _BaseCard<E extends React.ElementType = typeof BASE_CARD_DEFAULT_TAG>(
   forwardedRef: React.Ref<HTMLElement>,
 ) {
   const styles = getBaseCardStyles();
-  const [isFocused, setIsFocused] = useState(isFocusedProp ?? false);
```

Comment:
We can also remove the `isFocusProp` from the props list, if it's not used anywhere else.